src/partials/footer-content.hbs

```diff
@@ -73,7 +64,7 @@
         <li><a href="https://www.bonitasoft.com/downloads">Download</a></li>
         <li><a href="https://www.bonitasoft.com/events">Events</a></li>
         <li><a href="https://community.bonitasoft.com/">Community</a></li>
-        <li><a href="https://www.bonitasoft.com/sitemap">Site map</a></li>
+        <li><a href="https://bonitasoft.github.io/sitemap.xml">Site map</a></li>
```

Comment:
Replace` https://bonitasoft.github.io/sitemap.xml `by `{{ site.url }}/sitemap.xml` from template variable to be more generic